src/ControlSystem/ControlErrors/Size.hpp

```diff
@@ -189,8 +190,44 @@ struct Size : tt::ConformsTo<protocols::ControlError> {
         "TimescaleTuner for smoothing horizon measurements."};
   };
 
-  using options = tmpl::list<MaxNumTimesForZeroCrossingPredictor,
-                             SmoothAvgTimescaleFraction, SmootherTuner>;
+  struct State5Options {
```

Comment:
Up to you, but since internally this is called `DeltaRDriftOutward`, maybe name the option that too.  Seems more user-friendly.

Reply:
Oh yeah, probably a good idea. `5` is synonymous in my head with `DeltaRDriftOutward` which is why I put that